IbisLib/ibisapi.cpp

```diff
@@ -14,6 +14,8 @@
 #include "usacquisitionobject.h"
 #include "toolplugininterface.h"
 #include "objectplugininterface.h"
+#include "triplecutplaneobject.h"
```

Comment:
I think we want to keep these classes fairly isolated from the plugin api in order to be able to easily change the classes that implement the cut planes in the future. Also, IbisAPI shouldn't contain any processing. It should simply call Application and SceneManager functions.